src/hubspot-client.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
-    <Version>0.19.2</Version>
+    <Version>0.19.3</Version>
```

Comment:
**issue** we should keep version bumps out of MRs.

We handle that on master when releasing :)

Reply:
reverted